core/src/script/CGXP/plugins/Print.js

```diff
@@ -225,17 +249,91 @@ cgxp.plugins.Print = Ext.extend(gxp.plugins.Tool, {
      *        autoCreate: {tag: "input", type: "text", size: "45", maxLength: "45"}
      *    }]
      *
-     *  Default to: ``['title', 'comment', 'legend']``, that are predefined fields.
+     *  Default to: ``[]``.
      */
-    fields: ['title', 'comment', 'legend'],
+    fields: [],
+
+    /** api: config[options]
+     *  ``Object``
+     *  The options given to the print panel.
+     *  E.g.:
+     *
+     *  .. code:: javascript
+     *
+     *      options: {
+     *          labelAlign: "top",
+     *          defaults: {
+     *              anchor: "100%"
+     *          },
+     *          autoFit: true,
+     *          fieldsExtraClientConfiguration: {
+     *              "A4_portrait": {
+     *                  "title": {
+     *                      fieldAttributes: {
+     *                          fieldLabel: "${_("Name")}",
+     *                          emptyText: "${_("Name")}"
+     *                      }
+     *                  },
+     *                  "description": {
+     *                      useTextArea: true,
+     *                      fieldAttributes: {
+     *                          fieldLabel: "${_("Description")}",
+     *                          emptyText: "${_("Description")}",
+     *                          autoCreate: {maxLength: 200}
+     *                      }
+     *                  }
+     *              }
+     *          }
+     *      }
+     *
+     *  Default to: ``{}``.
+     */
+    options: {},
+
+    /** api: config[additionalAttributes]
+     *  ``Array``
+     *  Attributes added in the print form used especially with print V2.
+     *
+     *  Default to:
+     *
+     *  .. code:: javascript
+     *
+     *    [{
+     *        name: "Title",
```

Comment:
What about being consistent with the block at line 314:
* use lowercased strings for "name" ("title" instead of "Title")
* use capitalized first letter for "label" ("Title")
* use either "comment"/"Comment" or "comments"/"Comments", not a combination (sometimes plural, sometimes singular)
?